pom.xml

```diff
@@ -112,6 +113,53 @@
     <build>
         <plugins>
             <plugin>
+                <groupId>com.github.eirslett</groupId>
+                <artifactId>frontend-maven-plugin</artifactId>
+                <version>${frontend-maven-plugin.version}</version>
```

Comment:
nit: A good practice is to define a `<dependencyManagement>` in the parent pom to manage all dependencies in your maven project from one place and avoid different modules using different versions indadvertedly.

See for example the following depednency in a sub-module: https://github.com/okta/okta-core/blob/master/server/framework/pom.xml#L22

Where the definition comes from the parent: https://github.com/okta/okta-core/blob/master/pom.xml#L404

The way you are defining versions will work for now but, you might hit wall ahead on the road and pull your hair once your project scales.